app/controllers/admin/rdvs_controller.rb

```diff
@@ -94,20 +94,23 @@ def set_rdv
   end
 
   def rdv_params
-    params
-      .require(:rdv)
-      .permit(:motif_id, :status, :lieu_id, :duration_in_min, :starts_at, :context, :ignore_benign_errors, :max_participants_count, :name,
-              agent_ids: [],
-              user_ids: [],
-              rdvs_users_attributes: %i[user_id send_lifecycle_notifications send_reminder_notification id _destroy])
+    allowed_params = params.require(:rdv).permit(:motif_id, :status, :lieu_id, :duration_in_min, :starts_at, :context, :ignore_benign_errors, :max_participants_count, :name,
+                                                 agent_ids: [],
+                                                 user_ids: [],
+                                                 rdvs_users_attributes: %i[user_id send_lifecycle_notifications send_reminder_notification id _destroy],
+                                                 lieu_attributes: %i[name address latitude longitude])
+    return allowed_params.to_h.deep_merge(lieu_attributes: { organisation: current_organisation, availability: :single_use }) if params["rdv"]["lieu_attributes"].present?
```

Comment:
Que dirais-tu d'ajouter un commentaire pour expliquer pourquoi on édite les params ici ?

```suggestion
    # Quand un lieu ponctuel est saisi, il faut faire en sorte qu'il soit créé dans l'organisation courante.
    # Nous le faisons ici, côté serveur pour empêcher de spécifier une valeur arbitraire.
    if allowed_params[:lieu_attributes].present?
      allowed_params[:lieu_attributes][:organisation] = current_organisation
      allowed_params[:lieu_attributes][:availability] = :single_use
    end
```

J'ai aussi modifié la façon dont on modifie le hash pour qu'on ait pas besoin de comprendre comment `deep_merge` fonctionne.